manual/generator_activerecordcolumns.md

```diff
@@ -1,14 +1,14 @@
 ## ActiveRecordColumns
 
-`Tapioca::Compilers::Dsl::ActiveRecordColumns` refines RBI files for subclasses of `ActiveRecord::Base`
-(see https://api.rubyonrails.org/classes/ActiveRecord/Base.html). This generator is only
-responsible for defining the attribute methods that would be created for the columns that
-are defined in the Active Record model.
+`Tapioca::Compilers::Dsl::ActiveRecordColumns` refines RBI files for subclasses of
+[`ActiveRecord::Base`](https://api.rubyonrails.org/classes/ActiveRecord/Base.html).
+This generator is only responsible for defining the attribute methods that would be
+created for the columns that are defined in the Active Record model.
 
 **Note:** This generator, by default, generates weak signatures for column methods and treats each
```

Comment:
Should we add an example of this?

Reply:
I think giving an example here will complicate the documentation much more. On the other hand, I think we can bundle a version of that module with Tapioca (or in another gem named Tapioca Runtime, or something): https://github.com/Shopify/tapioca/issues/174#issuecomment-736867460